CHANGELOG.md

```diff
@@ -2,6 +2,22 @@
 
 **Note:** Before version 1.0.0, every release may contain breaking changes.
 
+## Version 0.12.0
+
+* **Breaking change**: Autofuzz API methods (`consume` and `autofuzz`) have moved from the
+  `Jazzer` class to the dedicated `Autofuzz` class
+* **Major feature**: Added JUnit 5 integration for fuzzing and regression tests using the
```

Comment:
Does not have to be included in the release but I would really like some further JUnit documentation to show this great feature.

Reply:
Yes, I will definitely add it as part of finishing the epic.